pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -153,4 +153,8 @@ message RunnerConfiguration {
   // auxiliary_metadata.
   map<string, buildbarn.resourceusage.MonetaryResourceUsage.Expense>
       costs_per_second = 10;
+
+  // Additional environment variables to set inside the runner.  These are
```

Comment:
Maybe add a comment there that even though this option is available, the use of action_env is preferred?

Reply:
Sure

Reply:
Great! Be sure to also put the link to the Bazel issue report in there, so people are aware of the limitations of action_env.